public/assets/sass/components/_table.scss

```diff
@@ -1,6 +1,219 @@
 .expenses {
     &__table {
-        justify-self: center;
-        width: 90%;
+        margin:0 20px ;
+        @include respond(phone){
+        // justify-self: center;
+        margin:0 ;
+        }
+
+        &--head {
+            display: grid;
+            grid-template-columns: repeat(3, minmax(4rem, 11rem)) minmax(min-content, 1fr) 8rem;
+            justify-content: center;
+            font-size: 1.6rem;
+            justify-items: center;
+            align-items: center;
+            @include respond(small-phone){
+                font-size: 1.4rem;
+            }
+            @include respond(phone) {
+             grid-template-columns: repeat(3, minmax(4rem , 11rem) ) 3rem ;
+             justify-content: space-between;
+             .comment-display-none {
+                 display: none;
+             }
+               
+            }
+            & > * {
+                padding: 3px;
+            }
+        }
+        &--row {
+            display: grid;
+            grid-template-columns: repeat(3, minmax(4rem, 11rem)) minmax(min-content,1fr) 8rem;
+            grid-template-rows: 4rem ;
+            justify-content: center;
+            border-radius: .4rem;
+            background-color: $black-table-row;
+            margin-bottom: .3rem;
+            padding: .2rem;
+            // .container input:checked ~ & {
+            //     grid-template-rows: 3rem  min-content;             
+            // }
```

Comment:
Kindly, remove unneeded snippets. If they are saved for future implementation, then add a TODO comment above the statement.